core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -421,9 +421,12 @@ public enum SqlKind {
   /** {@code DECODE} function (Oracle). */
   DECODE,
 
-  /** {@code NVL} function (Oracle). */
+  /** {@code NVL} function (Oracle, Spark). */
   NVL,
 
+  /** {@code NVL2} function (Oracle). */
```

Comment:
Are you saying that NVL2 doesn't exist on Spark?

Reply:
Sorry, I forgot to add

Reply:
done